product_docs/docs/transporter/ba_preview/getting_started/apply_constraints.mdx

```diff
@@ -2,4 +2,12 @@
 title: "Applying constraints"
 ---
 
-After the migration is completed, apply any existing constraints to the new database.
\ No newline at end of file
+Before starting the migration, you [prepared the schema](prepare_schema). Now, re-apply the constraints that were not included in the DDL import.
+
+## Primary Key and Unique constraints
+
+For **Primary Key** and **Unique** constraints, you have already created the tables and constraints in the target Postgres database, which means that the EDB Transporter has already mapped them and migrated data to the new instance. You don't need do to anything else. 
+
+## Other constraints 
+
+For constraints that are not of the type **Primary Key**, or **Unique**, you excluded them from the migration, which means you now have to manually re-apply them. Compare the target DDL objects with the source Oracle instance to ensure you don't forget to re-apply a constraint you excluded during the migration.
```

Comment:
Maybe we can provide a high-level overview of what it means to reapply the constraints. Does it mean to alter the DDL objects so they include the constraints? Would that happen one DDL object/query at the time?